### PR TITLE
Feature/display topics for users

### PR DESCRIPTION
**Pull request by lestairon**
## Description:

Create a controller to display all the available topics for the users

### CODE REVIEW

File: spec/requests/api/v1/topics_spec.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      let(:topics_number) { Faker::Number.between(from: 1, to: 100) }
+
+      before do
+        create_list(:topic, topics_number)
+        subject
+      end
+
+      it 'returns all persisted topics' do
+        expect(parsed_response.size).to eq(topics_number)
+      end
```
Comment (Ricoch) [inline spec/requests/api/v1/topics_spec.rb:19]: we should add a case in where we check the data returned by the controller (that it includes  `id`, `name` and `image_url`)

**Review by Ricoch — COMMENTED** [review]
🚀 
One comment about adding some more tests on the structure of the data returned, but it's looking great

File: spec/requests/api/v1/topics_spec.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      it 'returns the image_url in the body response' do
+        expect(parsed_response.first).to have_key('image_url')
+      end
+    end
+
+    context 'when there are no topics' do
+      before { subject }
+
+      it 'returns an empty array' do
+        expect(parsed_response).to eq([])
```
Comment (Ricoch) [inline spec/requests/api/v1/topics_spec.rb]: I think we can use `be_empty` here, can you try it out?
`expect(parsed_response).to be_empty`

**Review by Ricoch — APPROVED** [review]

File: spec/requests/api/v1/topics_spec.rb (hunk 2 above)
Comment (lestairon) [reply spec/requests/api/v1/topics_spec.rb]: True!

**Pull request merged**